storage/src/meta/mod.rs

```diff
@@ -658,6 +679,13 @@ impl BlobMetaState {
         let mut start = 0;
         let mut end = 0;
 
+        // FIXME: it's workaround for stargz chunks.
+        for i in 0..self.chunk_count {
```

Comment:
Seems a heavy overhead, we do need a flag for stargz blobs, otherwise we may run into trouble for normal rafs blobs.

Reply:
Fixed, checked stargz flag and added some comments.